### ISSUE TITLE
In flowjo: error "could not create gating - ML elements

### ISSUE THREAD

**Issue by MahoNagasawa** [tercen/flowjo_plugin#113]
in the full workflow template with 4 files concatenated file, Export was completed and a file was created in the exporting file, using the plugin function (import button) select the file, UMAP result ws loaded but not FlowSOM

![image](https://user-images.githubusercontent.com/87133370/174145706-e07f4ae9-24cf-4ff8-9a69-a81d38fdea99.png)

**Comment by ginberg** [comment]
did you try to restart FlowJo and start with a clean workspace? 
I can reproduce this error like this:
- choose to import the csv dataset -> all nodes will be created
- remove the nodes that were just created
- do the import again -> error like in your popup

It fails because of some caching that is done inside FlowJo

**Comment by MahoNagasawa** [comment]
I'm not quite sure what you mean, but what I did was (if I remember it correctly), first I loaded the quick workflow with the concatenated file (concat_1.fcs) upload in tercen, imported the result in FlowJo. that was only 1% processed data. I removed UMAP and FlowSOM result of those 1% data set, then imported the csv file that were created using Full template. could that be a reason?  to avoid this error or solve this error, what user need to do? because I can see people would try that, first with 1%, then want to see the full.

**Comment by ginberg** [comment]
yes, you did exactly what I described. For a next time, can you describe exactly the steps you did that lead to a certain outcome? So, it is easy to reproduce for someone else.

Restarting FlowJo solves the issue for me, did you try that as well? Maybe we should add that to the documentation, since it's nothing something that we can fix in the plugin
